### PR TITLE
Travis build support

### PR DESCRIPTION
**Pull request by hoffmanc**
I had to make a few minor changes to get this project to run the acceptance tests in Travis, mostly as a learning exercise.

The most interesting problem I saw was that **es6-promise**, despite being in the `dependencies` section of `client/package.json`, is not installed under `client/node_modules`, instead being nested under `client/node_modules/css-loader/node_modules/postcss/node_modules`, until I added a *duplicate* entry under `devDependencies`; I assume this phenomenon is easily explained, and merely reveals my nascence with node.js.

Thanks for taking a look.

### CODE REVIEW

**Comment by justin808** [comment]
@hoffmanc Why did you close this PR?

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -34,6 +34,8 @@
 RSpec.configure do |config|
   # config taken directly from RSpec example in the DatabaseCleaner README
   config.before(:suite) do
+    `npm install` or raise 'could not install deps'
```
Comment (justin808) [inline spec/rails_helper.rb]: @hoffmanc, this should be run after the deployment, and not as part of the suite.

File: client/package.json [hunk 2]
```diff
@@ -43,6 +43,7 @@
     "webpack": "^1.10.5"
   },
   "devDependencies": {
+    "es6-promise": "^2.3.0",
```
Comment (justin808) [inline client/package.json]: This should probably not be a dev dependency if it's needed as a shim.

**Comment by hoffmanc** [comment]
@justin808 sorry, still new to PRs - I didn't want to attach it to my forked **master**, since I wanted the ability to add more changes there.  I made a new PR that references a branch I made: #81

File: client/package.json (hunk 2 above)
Comment (hoffmanc) [inline client/package.json]: right, but not sure how to get es6-promise to be directly under node_modules without doing this (see original PR description)

File: spec/rails_helper.rb (hunk 1 above)
Comment (hoffmanc) [inline spec/rails_helper.rb]: @justin808 where should it live to get travis to run it during a build?

Comment (justin808) [inline spec/rails_helper.rb]: @hoffmanc I'm not sure about travis. We've been using CircleCi and CodeShip. @dylangrafmyre any opinion on this?

File: client/package.json (hunk 2 above)
Comment (justin808) [inline client/package.json]: `npm i --save es6-promise`

**Comment by samnang** [comment]
@hoffmanc no worry :) Next time better to create a feature branch for something that you want to work on, and then create pull request from that brench. Look at section Pull Request from here https://guides.github.com/activities/contributing-to-open-source/

Now, we have #80 and #81, so we better to close one and keep only one for discussions and latest changes.

**Comment by hoffmanc** [comment]
@samnang #81 is the one I created from a feature branch, and I already closed this one.

**Comment by samnang** [comment]
@hoffmanc :+1:

**Pull request closed without merging**